source/io.c

```diff
@@ -83,6 +83,18 @@ PyObject *aws_py_is_alpn_available(PyObject *self, PyObject *args) {
     return PyBool_FromLong(aws_tls_is_alpn_available());
 }
 
+PyObject *aws_py_is_tls_cipher_supported(PyObject *self, PyObject *args) {
+    (void)self;
+    (void)args;
+
+    int cipher_pref = 0;
+
+    if (!PyArg_ParseTuple(args, "i", &cipher_pref)) {
+        return PyErr_AwsLastError();
```

Comment:
```suggestion
        return NULL;
```

Py_ArgParseTuple() is a python-c-api function, it doesn't raise an AWS error

the way to report an exception in a python-c-api function is to set the current exception, and return NULL

PyArg_ParseTuple has already set the exception in this case, so we just need to return NULL